content/en/monitors/monitor_types/custom_check.md

```diff
@@ -16,28 +16,78 @@ further_reading:
 
 ## Overview
 
-Custom monitors encompass any service checks that are not reported by one of the out-of-the-box integrations included with the Agent.
+Custom check monitors include any service check not reported by one of the [350+ integrations][1] included with the Agent. Custom service checks can be sent to Datadog using a [custom Agent check][2], [DogStatsD][3], or the [API][4].
 
-Refer to the [Agent checks guide][1] for detailed information on writing your own checks that send metrics, events, or service checks.
+## Monitor creation
 
-## Configuration
+To create a [custom check monitor][5] in Datadog, use the main navigation: *Monitors --> New Monitor --> Custom Check*.
 
-1. Select your **custom check**:  
-  {{< img src="monitors/monitor_types/custom_check/Selecting_custom_check.png" alt="selecting custom monitor" responsive="true" style="width:80%;">}}
-2. Select **host or tags** that you would like to monitor.  
-  {{< img src="monitors/monitor_types/custom_check/monitor_scope.png" alt="monitor scope" responsive="true" style="width:80%;">}}
-  The check runs for every unique set of tags from all monitored hosts. For example, the `Nginx` service check reports one status per `{host,port}`. So if you have multiple servers running on a single host, then each one alerts separately in the case of failure.
+### Pick a Custom Check
 
-3. Select your **alert options**:  
-  {{< img src="monitors/monitor_types/custom_check/monitor_options.png" alt="monitor options" responsive="true" style="width:80%;">}}
-  While each check run sends a status of either CRITICAL, WARNING or OK, you can choose at what consecutive conditions to cause a state change and a notification. For example, you might want to know immediately when your check fails and only have it recover if it stays that way. In this case you might choose to notify on 1 critical status, 1 warning status and 4 OK statuses.  
-  You can optionally **notify on no data** after a configurable timeframe. You must choose at least 2 minutes for your timeframe.
+Choose a custom check from the drop-down box.
 
-4. Configure your **notification options**:  
-  Refer to the [Notifications][2] dedicated documentation page for a detailed walkthrough of the common notification options.
+### Pick monitor scope
+
+Select the scope to monitor by choosing host names, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```

Comment:
```suggestion
Select the scope to monitor by choosing hostnames, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```

seems like you're referring to the specific ones you choose in this instance. I will not fight you if you disagree though

Reply:
Checked with Cecilia on this. We are going to say this is referring to a group of hosts and not a particular (singular) host, so host names with a space.